### DISCUSSION TITLE
Paired Plot - Axis Limits

### DISCUSSION THREAD

**Discussion by timobage** [raphaelvallat/pingouin#407] [Q&A] [answered]
Hi,
how can I change the y-axis limits of the plot_paired() function?
If I use ax.set_ylim() it only shrinks/expands the existing plots but the limits of the underlying point and boxplot stay the same. I tried to give several different kwargs arguments but nothing worked.

Thank you for your help!

**Comment by raphaelvallat** [comment, marked as answer]
Hi,

Thanks for reporting this invalid behavior. It appears that the call to sns.despine(trim=True) in the plot_paired function is the culprit:

https://github.com/raphaelvallat/pingouin/blob/e6ca904e0392c23d0018f7ab4e5c4a690d6ec3ef/pingouin/plotting.py#L656

I'll remove it in a next version, but for now the easiest solution for you is to clone the repository locally, install Pingouin in develop mode (`python setup.py develop`) and then comment this line in the source code.

Thanks
Raphael
